packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -208,6 +226,24 @@ export default {
                 await this.$nextTick()
                 VueScrollTo.scrollTo(this.$refs.results, 400)
             }
+        },
+        handleDrop() {
+            this.dropbox.draggingOver = false
+        },
+        async dragOver() {
+            if (!this.dropbox.draggingOver) {
+                this.dropbox.draggingOver = true
+                this.dropbox.dragLeaveLocked = true
+
+                setTimeout(() => {
+                    this.dropbox.dragLeaveLocked = false
+                }, 100)
```

Comment:
Is it still necessary to lock the drag leave?

Reply:
Yes I tried it first without but still the same flickering behaviour